pkgs/development/libraries/kde-frameworks/baloo.nix

```diff
@@ -14,4 +14,12 @@ mkDerivation {
   ];
   outputs = [ "out" "dev" ];
   propagatedBuildInputs = [ kcoreaddons kfilemetadata qtbase ];
+
+  # kde-baloo.service uses `ExecCondition=@KDE_INSTALL_FULL_BINDIR@/kde-systemd-start-condition ...`
+  # which comes from the "plasma-workspace" derivation, but KDE_INSTALL_* all point at the "baloo" one
+  # (`${lib.getBin pkgs.plasma-workspace}` would cause infinite recursion)
+  postUnpack = ''
+    substituteInPlace "$sourceRoot"/src/file/kde-baloo.service.in \
+      --replace @KDE_INSTALL_FULL_BINDIR@ /run/current-system/sw/bin
```

Comment:
`/run/current-system/sw/bin` is an impurity. Please, replace it with `${plasma-workspace}/bin` instead

Reply:
While using `/run/current-system/sw/bin` is indeed impure, it's also highly unlikely that anyone is using KDE from nixpkgs outside of NixOS and I would personally much prefer a documented and known impurity if it fixes baloo right away rather trying to come up with a workaround.